ingress/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+-------------------------------------------------------------------------------
+
+LiveKit Ingress {{ .Values.image.tag | default .Chart.AppVersion }} has been deployed!
```

Comment:
It would be helpful to display text around any DNS mapping that they should perform. How do they get the IP address of the load balancer that's created? (a kubectl command to find out would be useful)

@biglittlebigben how does LiveKit server know what IP/URL to use for this ingress instance? when `CreateIngress`